<?xml version="1.0"?>
<launch>
    <!-- <node pkg="foresight" type="odom_to_pose.py" name="odom_to_pose"/> -->
    <include file="$(find foresight)/launch/localization/static_transformations.launch"/>
    <include file="$(find foresight)/launch/localization/px4flow_odom.launch"/>
    <include file="$(find foresight)/launch/localization/apriltags_tf.launch"/>
    <include file="$(find foresight)/launch/localization/ukf_fuser.launch"/>
</launch>
